docs/faq.md

```diff
@@ -0,0 +1,41 @@
+# Frequently asked questions
+
+### What's the difference between the available solvers in `conda`?
+
+Please refer to the section "Differences between `libmamba` and `classic`" in
+the [`libmamba-vs-classic`](./libmamba-vs-classic.md) docs.
+
+
+### How do I uninstall it?
+
+If you don't want to use the solver anymore, follow these instructions:
+
+1. If you configured it as the default solver, make sure you revert it with:
```

Comment:
If _not_ doing this step would break something, I would annotate with some sort of "⚠️ Important! ⚠️"-level of warning here.

Reply:
Done!